CODE_STYLE.md

```diff
@@ -0,0 +1,33 @@
+# Code Style
+
+Please follow this code style when contributing to `rwkv.cpp`.
+
+This list is not complete.
+
+## General
+
+Overall, keep code in similar style as it was before.
+
+- Keep lines at 180 characters or shorter.
+- Separate logically grouped pieces of code with empty lines.
+- Surround `if`, `for`, `while`, `do` and other similar statements with empty lines.
+- Write documentation for public functions indended for outside use.
+- Place single-line comments on the line before, not right after the code line.
+- Start comments with a capital letter, use correct grammar and punctuation.
+
+## C/C++
+
+- Use 4 spaces for indentation.
+- Place braces on the same line as the statement.
```

Comment:
this is known as [1TBS/OTBS (One True Brace Style)](https://en.wikipedia.org/wiki/Indentation_style#Variant:_1TBS_(OTBS))